ui/js/graphs.js

```diff
@@ -683,21 +684,22 @@ perf.controller('GraphsCtrl', [
               window.setTimeout(function () { modalInstance.updateTestInput(); }, 0);
             });
 
-            modalInstance.result.then(function(series) {
-              series.highlightedPoints = [];
-              series.visible = true;
-              series.color = availableColors.pop();
-
-              $scope.seriesList.push(series);
-              if( !$scope.highlightedRevision ) {
-                $scope.highlightedRevision = '';
+            modalInstance.result.then(function(seriesList) {
+              seriesList.forEach(function(series)  {
+                  series.hightlightedPoints = [];
+                  series.visible = true;
+                  series.color = availableColors.pop();
+                  $scope.seriesList.push(series);
+              });
+              if (!$scope.highlightedRevision) {
+                  $scope.highlightedRevision = '';                  
               }
               if (!$scope.zoom) {
-                $scope.zoom = {};
+                  $scope.zoom = {};
```

Comment:
So for now at least we have two space indentation inside this file. We want to switch to 4 to be consistent with the rest of treeherder eventually, but for now could you remove the extra whitespace (and ensure that new code is also indented 2 spaces)